alacritty/src/display/window.rs

```diff
@@ -496,3 +507,33 @@ fn use_srgb_color_space(window: &WinitWindow) {
         let _: () = msg_send![raw_window, setColorSpace: NSColorSpace::sRGBColorSpace(nil)];
     }
 }
+
+#[cfg(target_os = "macos")]
+fn set_background_opacity(window: &WinitWindow, opacity: CGFloat) {
+    let raw_window = match window.raw_window_handle() {
+        RawWindowHandle::AppKit(handle) => handle.ns_window as id,
+        _ => return,
+    };
+
+    // If transparent, fill background with NSColor.clearColor,
+    // otherwise fill with NSColor.windowBackgroundColor so that MacOS
+    // properly draws the window's border and opaque title bar.
```

Comment:
This sounds like the window's background is filled, but surely that's not what is happening, right? Just the titlebar background?